tinyproxy/start.sh

```diff
@@ -46,33 +46,34 @@ set_authentication()
 
 if [[ "${WEBPROXY_ENABLED}" = "true" ]]; then
 
-  echo "STARTING TINYPROXY"
+  echo "STARTING PRIVOXY"
 
-  find_proxy_conf
-  echo "Found config file $PROXY_CONF, updating settings."
+  # find_proxy_conf
+  # echo "Found config file $PROXY_CONF, updating settings."
 
-  set_port ${WEBPROXY_PORT} ${PROXY_CONF}
+  # set_port ${WEBPROXY_PORT} ${PROXY_CONF}
```

Comment:
configuration should still be possible, no?

Reply:
Fixed in https://github.com/haugene/docker-transmission-openvpn/pull/1760/commits/0e3391120b0898d3588138a3608264d8eed3b503